case-lib/opt.sh

```diff
@@ -3,28 +3,28 @@
 # Before using this, you must define these option arrays in your test
 # script. They must all be indexed by some unique, one-character
 # codename for each of your option.
-declare -A OPT_DESC OPT_NAME OPT_PARM_lst OPT_VALUE_lst
+declare -A OPT_DESC OPT_NAME OPT_HAS_ARG OPT_ARG
 
 # option setup && parse function
 func_opt_parse_option()
 {
     # OPT_NAME     (long) option name
     # OPT_DESC    short sentence describing the option
-    # OPT_PARM_lst    0 or 1: number of argument required
-    # OPT_VALUE_lst   default value overwritten by command line
-    #                 input if any. Set to 0 or 1 when PARM=0
+    # OPT_HAS_ARG    0 or 1: number of argument required
+    # OPT_ARG   default value overwritten by command line
+    #                 input if any. Set to 0 or 1 when OPT_HAS_ARG=0
 
     # for example
     # OPT_NAME['r']='remote'
     # OPT_DESC['r']='Run for the remote machine'
-    # OPT_PARM_lst['r']=1
-    # OPT_VALUE_lst['r']='' ## if PARM=1, must be set, if PARM=0, can ignore
+    # OPT_HAS_ARG['r']=1
+    # OPT_ARG['r']='' ## if OPT_HAS_ARG=1, must be set, if OPT_HAS_ARG=0, can ignore
```

Comment:
This comment is very misleading (I think it did misled you)

Reply:
Yes, I think I will delete the demo comment here sometime later. and as this PR require rebase, I will drop the comment fix commit. The clean up of opt.sh is just startup.